tests/execute/rsync/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartTest "Test with rsync removed during execute"
+        rlRun "tmt run -avvvddd \
+            provision -h virtual \
```

Comment:
Tests failed because of nested virtualization...

Reply:
yep, need to fix that

Reply:
@psss just do not know how? what do you suggest?

Reply:
adjust :) done :)

Reply:
The test ran the whole `tmt` test suite. Added tmp directory to run only the `rsync` removal command.